core/src/main/java/com/linecorp/armeria/common/AbstractRequestContext.java

```diff
@@ -151,8 +152,11 @@ public boolean isTimedOut() {
     /**
      * Marks this {@link RequestContext} as having been timed out. Any callbacks created with
      * {@code makeContextAware} that are run after this will be failed with {@link CancellationException}.
+     *
+     * @deprecated Use {@link DefaultServiceRequestContext#setRequestTimedOut()}.
      */
-    public void setTimedOut() {
+    @Deprecated
+    protected void setTimedOut() {
```

Comment:
Should still be public because otherwise it will be a breaking change.

Reply:
Got it. Reverted it. Thanks!